client/keys/util.go

```diff
@@ -51,3 +60,72 @@ func SaveKeyList(path string, keyList map[string]Key) error {
 
 	return nil
 }
+
+func GetKeyToSign(path string) (Key, error) {
+	var key = Key{}
+
+	keyList, err := LoadKeyList(path)
+	if err != nil {
+		return key, err
+	}
+
+	switch len(keyList) {
+	case 0:
+		return key, errors.New("Keys are not found on local storage")
+	case 1:
+		// safe to use for loop to find the first value of map
+		for _, value := range keyList {
+			key = value
+		}
+		return key, nil
+	}
+
+	err = List(path)
+	if err != nil {
+		return key, nil
+	}
+
+	reader := bufio.NewReader(os.Stdin)
+
+	fmt.Printf("\nType the nickname corresponding to the key to sign this tx: ")
+	nickname, err := reader.ReadString('\n')
+	if err != nil {
+		return key, err
+	}
+
+	nickname = strings.Replace(nickname, "\r\n", "", -1)
+	nickname = strings.Trim(nickname, "\n")
+
+	keyStatus := Check(nickname, path)
+	switch keyStatus {
+	case NoExists:
+		return key, errors.New("The key doesn't exist")
+	}
+
+	key = keyList[nickname]
+
+	return key, nil
+}
+
+func GetDecryptedKey(key *Key) error {
```

Comment:
this function "transforms" the key that we already have. what about renaming it to "decrypt key".